pkg/collector/corechecks/embed/process_agent.go

```diff
@@ -131,7 +131,7 @@ func (c *ProcessAgentCheck) run() error {
 func (c *ProcessAgentCheck) Configure(data integration.Data, initConfig integration.Data, source string) error {
 	// only log whether process check is enabled or not but don't return early, because we still need to initialize "binPath", "source" and
 	// start up process-agent. Ultimately it's up to process-agent to decide whether to run or not based on the config
-	if enabled := config.Datadog.GetBool("process_config.enabled"); !enabled {
+	if enabled := config.GetProcessCollectionEnabled(config.Datadog); !enabled {
```

Comment:
Hi, thank you for your suggestion. 

Currently, the reason we don't have `config.Datadog` as a receiver is because it makes it easy to test using configs other than `config.Datadog`, and even gives us the flexibility to use a mock config should we choose to do so. Additionally, since `config.Datadog` uses the `config.Config` interface, we would have to re implement this method for every struct that implements the `config.Config` interface, which may be a lot of unnecessary code duplication.

Also when you say do everything include the log.info below, do you mean that `LaunchProcessCollection` would also launch the process agent, or just display a message saying that process collection is disabled? Our goal here is only to add a helper function that displays a warning if a deprecated setting is used, and let `cmd/process-agent` only have to check the global config to see if the process or container checks are enabled